clouds/clouds-core/src/test/java/sunstone/core/setup/SetupSuite.java

```diff
@@ -0,0 +1,11 @@
+package sunstone.core.setup;
+
+import org.junit.platform.suite.api.SelectClasses;
+import org.junit.platform.suite.api.Suite;
+import sunstone.core.setup.suitetests.SetupCleanupSecondTest;
+import sunstone.core.setup.suitetests.SetupFirstTest;
+
+@Suite
+@SelectClasses({SetupFirstTest.class, SetupCleanupSecondTest.class})
```

Comment:
I'm not sure whether ordering is guarunteed by JUnit 5.

Reply:
It is not stated anywhere, but it is an array, hence I assume why the order is the same. But I agree this is not ideal. Here is the issue. https://github.com/wildfly-extras/sunstone/issues/126